cli/.js/src/main/scala/aqua/ArgGetterService.scala

```diff
@@ -0,0 +1,43 @@
+package aqua
+
+import aqua.ArgGetterService.GETTER_SERVICE_ID
+import aqua.js.{CallJsFunction, CallServiceHandler, FluencePeer}
+import aqua.model.func.Call
+import aqua.model.{LiteralModel, VarModel}
+import aqua.model.func.raw.CallServiceTag
+
+import scala.concurrent.Promise
+
+// Service that can return argument to use it from a code
+case class ArgGetterService(value: VarModel, fnName: String, arg: scalajs.js.Dynamic) {
```

Comment:
Why do you need a separate `fnName` argument? All info is already in `value: VarModel`